utils_nlp/models/transformers/sequence_classification.py

```diff
@@ -81,6 +96,9 @@ def __init__(
         self.seed = seed
         self.fp16 = fp16
 
+    def list_supported_models():
```

Comment:
@classmethod decorator?

Reply:
Yeah this can be a decorator `@classmethod` sounds about right